### DISCUSSION TITLE
Compiling Rustdesk client for Win via github is not signed

### DISCUSSION THREAD

**Discussion by xarlien** [rustdesk/rustdesk#7069] [General]
Hello, when I try compile by github actions from repository, EXE for Windows is not signed. Downloaded EXE from web rustdesk.com is signed. It is possible compile from github repository with right signing (work with smartscreen filter in Win)?

**Comment by xlionjuan** [comment]
Buy certificate, 270USD/year

**Comment by xarlien** [comment]
I want to use opensource because I don't want to pay for overpriced teamvieewer/anydesk licenses. For the price of 270USD I have a commercial license Anydesk for a full year. There must be some other way.

**Reply by Karbust** [reply]
https://github.com/rustdesk/rustdesk/discussions/6467

Unless RustDesk's team decides to use another CA that provides Cloud HSMs without the need for key attestation (the likes of DigiCert), it won't be happening.

Another option, is what I wrote on my comment, we need to see what RustDesk's team decides to do. https://github.com/rustdesk/rustdesk/discussions/6467#discussioncomment-8334387
